SearchHistory/models.py

```diff
@@ -1,22 +1,24 @@
 from django.db import models
 from django.utils import timezone
+from account.models.professional import Professional
+from account.models.client import Client
 
 
 class SearchHistory(models.Model):
     search_id = models.BigAutoField(primary_key=True)
-    professional_id = models.PositiveIntegerField(null=False, blank=False)
-    client_id = models.PositiveIntegerField(null=False, blank=False)
+    professional_id = models.ForeignKey(Professional, on_delete=models.CASCADE)
+    client_id = models.ForeignKey(Client, on_delete=models.CASCADE)
     date = models.DateTimeField(default=timezone.now)
 
     class Meta:
         db_table = 'SearchHistory'
 
+    def __str__(self):
+        return f"{self.client_id} search {self.professional_id} [{self.date}]"
+
     @staticmethod
-    def get_5_last_professionals_search_by_client(client_id: int):
+    def get_5_last_professionals_search_by_client(client_id):
         lst = SearchHistory.objects.filter(client_id=client_id)
         lst = lst.order_by('-date')[:5]
```

Comment:
Note that you assume here that expected_result size is bigger than 5...
What happens in the case that client has not yet searched for 5 professionals? (The size of expected_result is smaller than 5)

Reply:
@OfirBachar ??

Reply:
done